do/zones.go

```diff
@@ -55,10 +55,11 @@ func (z zones) GetZoneByProviderID(providerID string) (cloudprovider.Zone, error
 	return cloudprovider.Zone{Region: d.Region.Slug}, nil
 }
 
-// GetZoneByNodeName returns the Zone containing the current zone and locality
-// region of the node specified by node name. GetZoneByNodeName will only fill
-// the Region field of cloudprovider.Zone for DO.
-func (z zones) GetZoneByNodeName(nodeName types.NodeName) (cloudprovider.Zone, error) {
+// GetZoneByNodeName returns a cloudprovider.Zone from the droplet identified
+// by nodeName. GetZoneByNodeName only sets the Region field of the returned
+// cloudprovider.Zone.
+func (z zones) GetZoneByNodeName(nodeName types.NodeName) (cloudprovider.Zone,
+	error) {
```

Comment:
This line break was intentional right?

Reply:
No. It would work as it was, but it wasn't intentional. I unwrapped it.